progressReport/canvasEmbedding.js

```diff
@@ -0,0 +1,52 @@
+const currentUrl = window.location.href;
+
+
+
+const targetPageUrl = 'https://ucberkeleysandbox.instructure.com/courses/141/pages/concept-map?module_item_id=418';
+const alternateTargetPageUrl = 'https://ucberkeleysandbox.instructure.com/courses/141/pages/concept-map';
+// Paste Canvas API key here
+const accessToken = '';
```

Comment:
Whose access token is this?

Reply:
We discussed not using student access tokens, so if this is a service account access token, that's fine, but only if the students cannot see this js.  How is this handled?

Reply:
This is a service account access token. Students will not be able to see this js; the Canvas Theme Editor allows this code to be run server-side.